test/src/pages.js

```diff
@@ -13,6 +15,9 @@ class Pages {
         const WelcomePage = require(`./pages/${platform}/welcome.page`);
         Pages.welcomePage = new WelcomePage();
 
+        const BasePage = require(`./pages/base.page`);
```

Comment:
platform param needed in order to avoid violating the current pageObject structure

Reply:
For this to happen, I will have to create two parent classes within each ios and android package and you would be duplicating everything.